crates/ruma-events/src/sticker.rs

```diff
@@ -26,11 +31,142 @@ pub struct StickerEventContent {
 
     /// The URL to the sticker image.
     pub url: OwnedMxcUri,
+
+    /// The media source
+    #[cfg(not(feature = "compat-encrypted-stickers"))]
+    #[serde(skip)]
+    pub source: MediaSource,
```

Comment:
This is not what I meant. Since `MediaSource` also contains the `url` field in theory, it should also replace the `url` field.

But since we want the `Encrypted` variant to not be available by default, we should use an almost identical type, with the only difference that the `Encrypted` variant is behind the cargo feature.